spec/services/ndelius/fake_record_spec.rb

```diff
@@ -55,7 +55,11 @@
     context 'when the Nomis id ends with Q, R, S, T' do
       it 'raises a NoTier exception' do
```

Comment:
We are not expecting an exception any more

Reply:
fixed,